build/azDevOps/azure/azure-pipelines-javaspring-k8s.yml

```diff
@@ -399,7 +399,7 @@ stages:
           - name: app_insights_instrumentation_key
             value: "$[ dependencies.AppInfraDev.outputs['AppInfraDev.tfoutputs.app_insights_instrumentation_key'] ]"
           - name: namespace
-            value: "$(Environment.ShortName)-java-api-cqrs-events"
+            value: "$(Environment.ShortName)-${{ variables.domain }}"
```

Comment:
where's `variables.domain` coming from? We should update the other projects like so

Reply:
It is set as a variable in the build pipeline file on line 50